.github/workflows/main.yml

```
@@ -105,6 +106,8 @@ jobs:
             echo "## Install"
             echo "1. Download the rock."
             echo "2. Install it using luarock: \`luarock install <ROCK>.src.rock\`"
+            echo ""
+            echo "**Full Changelog**: https://github.com/DataDog/kong-plugin-ddtrace/compare/v{LATEST_RELEASE}...${GITHUB_SHA}"
```

```suggestion
            echo "**Full Changelog**: https://github.com/DataDog/kong-plugin-ddtrace/compare/v${LATEST_RELEASE}...${GITHUB_SHA}"
```

I think you should also be able to remove cutting the `v` character when assigning the variable, and therefore omitting it in this line